linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_for_su/rule.yml

```diff
@@ -22,6 +22,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1.1
+    srg: 'SRG-OS-000373-GPOS-00156,SRG-OS-000312-GPOS-00123'
```

Comment:
why the single quotes?

Reply:
You are right, they are not needed. I've removed them.